packages/components/src/components/icon/library.system.ts

```diff
@@ -17,6 +17,15 @@ export const icons = {
     <svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
       <path d="M12.743 6.333c-0.188-0.195-0.452-0.316-0.743-0.316s-0.555 0.121-0.743 0.315l-0 0-9 10c-0.137 0.17-0.22 0.388-0.22 0.626 0 0.552 0.448 1 1 1 0.276 0 0.526-0.112 0.707-0.293v0l8.257-9.173 8.257 9.173c0.181 0.181 0.431 0.292 0.707 0.292 0.552 0 1-0.448 1-1 0-0.238-0.083-0.456-0.222-0.628l0.001 0.002z"></path>
     </svg>
+  `,
+  close: `
+  <svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+    <path d="M21.707 2.293a1 1 0 0 0-1.414 0L12 10.586 3.707 2.293a1 1 0 0 0-1.413 1.414l-.001-.001 8.293 8.293-8.293 8.293a1 1 0 0 0 0 1.414 1 1 0 0 0 1.414 0L12 13.413l8.293 8.293a1 1 0 0 0 1.414 0 1 1 0 0 0 0-1.414l-8.293-8.293 8.293-8.293a1 1 0 0 0 0-1.414z"/></svg>
+  </svg>
+`,
+  'arrow-left': `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <path d="M21 10.998H5.413l5.293-5.293A1 1 0 1 0 9.293 4.29l-7 7a1 1 0 0 0 0 1.414l7 7a1 1 0 0 0 1.414-1.414L5.412 13H21a1 1 0 0 0 0-2v-.001Z"/>
+</svg>
```

Comment:
This is not something we need in the component, as we just use that in the sample stories. You should use `<sd-icon library="global-resources" icon="arrow-left"></sd-icon>` there.

Reminder: System Icons are for Icons, that have to be built-in in our design ssstem, without having to rely on a CDN. Mostly this is only for itnernal use. Everything else has to be outside.